docs/modules/ROOT/pages/how-to-guides/proc_managing-compliance-with-the-enterprise-contract.adoc

```diff
@@ -18,7 +18,9 @@ The Enterprise Contract (EC) is an artifact verifier and customizable policy che
 . Select *Add integration test*.
 . In the *Integration test name* field, enter a name of your choosing.
 . In the *GitHub URL* field, enter *https://github.com/redhat-appstudio/build-definitions*
-. In the *Path in repository* field, enter */pipelines/enterprise-contract.yaml*
+. In the *Path in repository* field, to use the default EC configuration, enter */pipelines/enterprise-contract.yaml*
+.. You can also enter any of the paths listed link:https://github.com/enterprise-contract/config#readme[here], to use a configuration that matches your specific needs more closely.
```

Comment:
.. You can also enter any of the paths listed in the link:https://github.com/enterprise-contract/config#readme[Enterprise Contract Configuration Files], to use a configuration that matches your specific needs.